org.metafacture.fix/src/test/java/org/metafacture/metafix/MetafixRecordTest.java

```diff
@@ -0,0 +1,703 @@
+/*
+ * Copyright 2021 Fabian Steeg, hbz
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.metafacture.metafix;
+
+import org.metafacture.framework.StreamReceiver;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+
+/**
+ * Tests Metafix record level methods. Following the cheat sheet
+ * examples at https://github.com/LibreCat/Catmandu/wiki/Fixes-Cheat-Sheet
+ *
+ * @author Fabian Steeg
+ */
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("checkstyle:MultipleStringLiterals")
```

Comment:
Suggesting switching to `@SuppressWarnings` in https://github.com/metafacture/metafacture-fix/pull/60/commits/ea54a0ec96a2bb90a47f057f70c05c83d563fd82, as it's the standard Java suppression mechanism.

Reply:
But it's also limited in where it can appear! Now the check is suppressed in the whole method `org.metafacture.metafix.InterpreterTest.shouldInterpretNested()`; should we - unintentionally - add another "magic number", we wouldn't notice.

Also:

1. At least in our case, check violations are errors, not warnings.
2. We can't limit the number of allowed exceptions (currently two).
3. The `SuppressWithNearbyCommentFilter` should be removed if we're no longer using it.

Reply:
Opened https://github.com/metafacture/metafacture-fix/issues/62 to follow up on this.